lib/marks.rb

```diff
@@ -0,0 +1,17 @@
+module Marks
+
+  X = "X"
+  O = "O"
+
+  COMPUTER_MARK = O
+  OPPONENT_MARK = X
```

Comment:
Should this information `computer vs opponent` be present here?
I shouldn't matter if it's a human or a computer that use a mark or another

Reply:
It looks like you are just using these constants in tests - if so, you could declare them in the test file if they bring more clarity to your test case and remove them from this Marks file.

Reply:
I'm also using them in the new class ScoreCalculator and in UnbeatableComputer. I just thought to make that distinction in Marks because Marks::COMPUTER_MARK seemed clearer than Marks::O (which is the mark for computer?) But I can change them, if you think